content/blog/2021/09/running-rego-policies.md

```diff
@@ -0,0 +1,141 @@
+organization establishes over time.
+
+## Keep generic generic
+
+As a generic platform, we are interested in supporting as many
+languages and frameworks as possible. Currently, we have SDK's for
+Rust, Go and Swift. The only requirement we have to introduce other
+languages at the moment is that there is a waPC backend for that
+language, so the host (Kubewarden) can communicate with the guest
+(your policy) using waPC. This interface might change in the future,
+but that matter is out of the scope of this post.
+
+If you are familiar with Kubewarden, you might know that it provides a
+WebAssembly runtime in order to execute policies and let the policies
+do the heavy lifting of enforcing certain attributes, properties and
+operations on your Kubernetes resources.
+
+## Rego
+
+Rego is a programming language created with a main purpose in mind:
+perform queries on an input, and provide an output in response.
+
+Rego is a declarative language highly optimized for making reading and
+writing policies as easy as possible. It being a different and unknown
+programming language has a learning curve, as all programming
+languages do.
+
+### Stronger together
+
+Open Policy Agent allows to compile a policy to a Web Assembly object
+file. This makes a strong foundation for running Rego policies in
+Kubewarden.
+
+By doing so, we are able to reuse on Kubewarden all the policies we
+have already written in Rego.
+
+#### One language, two frameworks
+
+Rego is used by Open Policy Agent and Gatekeeper projects. There are
+some differences between both, but one important takeaway is that the
+language does not differ. It is exactly the same. The way input,
+parameters, and other important information is exposed to the policy,
+and the way the policy has to give its answer differs between them,
+but the differences end there.
+
+By targeting Web Assembly, with the `opa build` command, it's possible
+to build policies targeting Open Policy Agent and Gatekeeper in a
+format that is understood and runnable by Kubewarden: `wasm`.
+
+Having presented the context, we can now go for the good news:
+Kubewarden supports both **Open Policy Agent** and **Gatekeeper**
+policies! -- **without any change needed on your existing policies**.
+
+#### A burrito, with some Rego inside of it
+
+[Flavio](https://twitter.com/flavio_castelli) had the idea and did the
+initial work of implementing a runtime for Rego policies built for the
+Web Assembly target inside Kubewarden. As you can imagine, the only
+logical name one can think of for a Burrito full of Rego can be
+`burrego`. This project is currently embedded into our
+`policy-evaluator` project in a crate of its own. If there is
+community interest, we can publish it as an individual crate on
+`crates.io`, so let us know if this is of your interest.
+
+## Distributing policies
+
+### Building policies
+
+Despite your policy code does not need to be adapted at all, the first
```

Comment:
Nit:

```suggestion
Despite your policy code not needing to be adapted at all, the first
```

(But maybe it would be better simplified?)

Reply:
I am not sure about this one, I think the original is better formed, but not a native speaker.